crates/next-core/src/next_font/google/util.rs

```diff
@@ -114,48 +152,66 @@ pub(super) fn get_stylesheet_url(
 ) -> Result<String> {
     // Variants are all combinations of weight and style, each variant will result
     // in a separate font file
-    let mut variants: Vec<Vec<(&str, &str)>> = vec![];
-    if axes.wght.is_empty() {
+    let mut variants: Vec<Vec<(&str, VariantValue)>> = vec![];
+
+    let weights = match &axes.wght {
+        FontAxesWeights::Variable(Some(wght)) => {
+            vec![VariantValue::String(wght.to_owned())]
+        }
+        FontAxesWeights::Variable(None) => {
+            vec![]
+        }
+        FontAxesWeights::Fixed(wghts) => {
+            let mut wghts = wghts.clone();
+            wghts.sort();
```

Comment:
A bit out of scope for this PR maybe, but I'd personally prefer to do the sorting upon construction of `FontAxesWeights` (with a constructor method), because that could be theoretically be better for turbo-tasks caching.